contracts/HydraDelegation/modules/VestedDelegation/VestedDelegation.sol

```diff
@@ -51,14 +48,14 @@ contract VestedDelegation is
 
     // _______________ Initializer _______________
 
-    function __VestedDelegation_init(address _vestingManagerFactoryAddr, address _hydraChainAddr) internal onlyInitializing {
+    function __VestedDelegation_init(
+        address _vestingManagerFactoryAddr,
+        address _hydraChainAddr,
+        address _rewardWalletAddr
+    ) internal onlyInitializing {
         __VestingManagerFactoryConnector_init(_vestingManagerFactoryAddr);
         __HydraChainConnector_init(_hydraChainAddr);
-        __VestedDelegation_init_unchained();
-    }
-
-    function __VestedDelegation_init_unchained() internal onlyInitializing {
-        balanceChangeThreshold = 32;
+        __RewardWalletConnector_init(_rewardWalletAddr);
```

Comment:
Why RewardWalletConnector is initialized in the vested delegation when it is used in the base Delegation as well

Reply:
Well, it will be better to be initialized in the Delegation. I was just mistaken by the HydraChainConnector, but it is used only in the VestedDelegation.